### DISCUSSION TITLE
How to turn off the view transition?

### DISCUSSION THREAD

**Discussion by EdwardDeaver** [satnaing/astro-paper#401] [Q&A]
Hi, 

Love the theme but how do I turn off the view transition that moved the type when you click, it's distracting for me.

**Comment by patrick-kot** [comment]
inside `src\layouts\Layout.astro` just comment or delete `<ViewTransitions />` tag

**Comment by satnaing** [comment]
Hello @EdwardDeaver,
Like @patrick-kot said, removing `<ViewTransitions />` will solve your issue.

However, after removing `<ViewTransitions />`, you'll see a white flash if you navigate to another page. 
So, I suggest you to do the following

file: `src/layouts/Layout.astro`

```html
<!-- Remove this -->
<ViewTransitions />

<!-- Remove async -->
<script is:inline src="/toggle-theme.js"></script>
```

Because of the above changes, the issue I mentioned will be gone.
However, as a trade-off, the site will have an insignificant point deduction in Google's core web vitals score.
